[2024-04-15T12:45:31.760+0000] {processor.py:161} INFO - Started process (PID=2990) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:45:31.762+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:45:31.763+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:45:31.762+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:45:31.810+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:45:31.799+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline_dag.py", line 6, in <module>
    from tasks.extract.extract_data import extract_data
ModuleNotFoundError: No module named 'tasks'
[2024-04-15T12:45:31.813+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:45:31.839+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.086 seconds
[2024-04-15T12:46:02.392+0000] {processor.py:161} INFO - Started process (PID=3049) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:46:02.394+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:46:02.395+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:46:02.395+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:46:02.411+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:46:02.407+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline_dag.py", line 6, in <module>
    from tasks.extract.extract_data import extract_data
ModuleNotFoundError: No module named 'tasks'
[2024-04-15T12:46:02.413+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:46:02.443+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.058 seconds
[2024-04-15T12:46:33.077+0000] {processor.py:161} INFO - Started process (PID=3108) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:46:33.093+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:46:33.095+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:46:33.094+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:46:33.133+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:46:33.129+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline_dag.py", line 6, in <module>
    from tasks.extract.extract_data import extract_data
ModuleNotFoundError: No module named 'tasks'
[2024-04-15T12:46:33.136+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:46:33.164+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.105 seconds
[2024-04-15T12:47:03.661+0000] {processor.py:161} INFO - Started process (PID=3167) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:03.663+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:47:03.664+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:03.664+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:03.685+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:03.681+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline_dag.py", line 6, in <module>
    from tasks.extract.extract_data import extract_data
ModuleNotFoundError: No module named 'tasks'
[2024-04-15T12:47:03.686+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:03.712+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.058 seconds
[2024-04-15T12:47:34.365+0000] {processor.py:161} INFO - Started process (PID=3226) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:34.366+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:47:34.367+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:34.367+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:34.386+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:34.383+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/etl_pipeline_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_pipeline_dag.py", line 6, in <module>
    from tasks.extract.extract_data import extract_data
ModuleNotFoundError: No module named 'tasks'
[2024-04-15T12:47:34.388+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:34.410+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.053 seconds
[2024-04-15T12:47:36.814+0000] {processor.py:161} INFO - Started process (PID=3227) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:36.816+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:47:36.817+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:36.817+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:36.882+0000] {processor.py:840} INFO - DAG(s) 'etl_pipeline_dag' retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:47:37.142+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:37.141+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:etl_pipeline_dag
[2024-04-15T12:47:37.158+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:37.158+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:etl_pipeline_dag
[2024-04-15T12:47:37.169+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:37.168+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:etl_pipeline_dag
[2024-04-15T12:47:37.170+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:37.169+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-15T12:47:37.186+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:37.186+0000] {dag.py:3118} INFO - Creating ORM DAG for etl_pipeline_dag
[2024-04-15T12:47:37.189+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:47:37.187+0000] {dag.py:3954} INFO - Setting next_dagrun for etl_pipeline_dag to None, run_after=None
[2024-04-15T12:47:37.225+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.424 seconds
[2024-04-15T12:48:07.692+0000] {processor.py:161} INFO - Started process (PID=3286) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:48:07.695+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:48:07.697+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:48:07.697+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:48:07.873+0000] {processor.py:840} INFO - DAG(s) 'etl_pipeline_dag' retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:48:07.933+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:48:07.933+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-15T12:48:08.010+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:48:08.010+0000] {dag.py:3954} INFO - Setting next_dagrun for etl_pipeline_dag to None, run_after=None
[2024-04-15T12:48:08.062+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.389 seconds
[2024-04-15T12:48:39.979+0000] {processor.py:161} INFO - Started process (PID=3345) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:48:39.980+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:48:39.981+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:48:39.981+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:48:40.005+0000] {processor.py:840} INFO - DAG(s) 'etl_pipeline_dag' retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:48:40.028+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:48:40.028+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-15T12:48:40.074+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:48:40.072+0000] {dag.py:3954} INFO - Setting next_dagrun for etl_pipeline_dag to None, run_after=None
[2024-04-15T12:48:40.158+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.187 seconds
[2024-04-15T12:49:10.718+0000] {processor.py:161} INFO - Started process (PID=3399) to work on /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:49:10.720+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/etl_pipeline_dag.py for tasks to queue
[2024-04-15T12:49:10.721+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:49:10.721+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:49:10.744+0000] {processor.py:840} INFO - DAG(s) 'etl_pipeline_dag' retrieved from /opt/airflow/dags/etl_pipeline_dag.py
[2024-04-15T12:49:10.779+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:49:10.775+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-04-15T12:49:10.806+0000] {logging_mixin.py:188} INFO - [2024-04-15T12:49:10.806+0000] {dag.py:3954} INFO - Setting next_dagrun for etl_pipeline_dag to None, run_after=None
[2024-04-15T12:49:10.883+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/etl_pipeline_dag.py took 0.173 seconds
